{"name": "Проверка появления ошибки при создании курьера с таким же логином", "status": "passed", "description": "Тестирование класса создания курьера", "steps": [{"name": "Создание курьера с уникальными данными", "status": "passed", "start": 1751995494974, "stop": 1751995495139}, {"name": "Создание курьера с существующим логином", "status": "passed", "start": 1751995495139, "stop": 1751995495262}], "start": 1751995494974, "stop": 1751995495262, "uuid": "4a8128b2-83fd-46f4-a849-634217a20ace", "historyId": "b02b4756866b7b55276fa72c2a32bba6", "testCaseId": "b02b4756866b7b55276fa72c2a32bba6", "fullName": "tests.test_create_courier.TestCreateCourier#test_error_when_creating_courier_with_existing_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "DESKTOP-PM8THT8"}, {"name": "thread", "value": "10716-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}